test/e2e-tests.sh

```diff
@@ -34,15 +34,7 @@ else
   echo "BROKER_CLASS is set to '${BROKER_CLASS}'. Running tests for that broker class."
 fi
 
-if [ "${BROKER_CLASS}" == "KafkaNamespaced" ]; then
-  # if flag exists, only test tests that are relevant to namespaced KafkaBroker
-  echo "BROKER_CLASS is set to 'KafkaNamespaced'. Only running the relevant tests."
-  go_test_e2e -timeout=1h ./test/e2e_broker/... || fail_test "E2E suite failed (directory: ./e2e_broker/...)"
-  success
-fi
```

Comment:
This means we can remove the separate `integration-tests-namespaced-broker_eventing-kafka-broker_main` job

Reply:
Test infra config is at https://github.com/knative/test-infra/blob/main/prow/jobs_config/knative-sandbox/eventing-kafka-broker.yaml

Reply:
As I understood yes. This should be now covered by the `reconciler-tests-namespaced-broker` job.
@aliok: any protest?

Reply:
PR: https://github.com/knative/test-infra/pull/3758